### DISCUSSION TITLE
What is the best alternative to eSpeak?

### DISCUSSION THREAD

**Discussion by cod3r0k** [idiap/coqui-ai-TTS#118] [Q&A]
I want to deploy a high-quality TTS system. Could you suggest the best alternative to eSpeak?

**Comment by eginhard** [comment]
Why not Espeak? Any phonemizer that supports your target language should be fine, or just use graphemes.

**Reply by cod3r0k** [reply]
Dear @eginhard My concern is related to another issue I mentioned before: handling mixed-language text input.

**Reply by eginhard** [reply]
Ok, please always provide the full context for a question, so that it is clear. As mentioned in #115 you could detect the language of each word and then call your phonemizer separately per language, or just use graphemes.
